### DISCUSSION TITLE
Button with fontawesome icon possible?

### DISCUSSION THREAD

**Discussion by dminca** [jpanther/congo#273] [Q&A] [answered]
Is it possible to add a button with a fontawesome icon?

I've found [this article](https://mcshelby.github.io/hugo-theme-relearn/shortcodes/button/) which shows it's possible, but I tried that approach and unfortunately the icon was not rendering for me.

```mustache
{{% button href="https://gohugo.io/" %}}Get Hugo{{% /button %}}
{{% button href="https://gohugo.io/" style="warning" icon="dragon" %}}Get Hugo{{% /button %}}
```

<img width="235" alt="image" src="https://user-images.githubusercontent.com/5355219/184452316-e59b3069-e873-46dc-a511-66ccb2ee0f90.png">

**Comment by jpanther** [comment, marked as answer]
The article you're reading is from a different theme and the button shortcode in Congo doesn't have an `icon` parameter. The theme does however support this use case by using the `icon` shortcode inside the button content.

If you use the following syntax you can add SVG icons into a button:

```html
{{< button href="#button" >}}
{{< icon "github" >}} Check out my GitHub
{{< /button >}}
```

Any of the icons bundled in the theme are available, as well as support to add your own. Check out the [docs](https://jpanther.github.io/congo/docs/shortcodes/#icon) for more details.

**Reply by dminca** [reply]
Thanks James, that works really good! :)
